src/pages/[special].astro

```
@@ -0,0 +1,47 @@
+---
+import DefaultLayout from '@layouts/DefaultLayout.astro'
+import PageHeader from '@components/PageHeader.astro'
+import SocialShares from '@components/SocialShares.astro'
```

ESLint

[error] 'SocialShares' is defined but never used.

(no-unused-vars)